dts/arm/st/h7/stm32h747.dtsi

```diff
@@ -4,4 +4,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include <st/h7/stm32h7.dtsi>
+#include <st/h7/stm32h7_dualcore.dtsi>
+
+/ {
+	cpus {
+		cpu@0 {
+			compatible = "arm,cortex-m7";
+		};
+
+		cpu@1 {
+			compatible = "arm,cortex-m4f";
+		};
+	};
+};
```

Comment:
I still don't understand why these lines (9 to 19) are needed.
Please remove

Reply:
I remove these line.